src/glyphs.js

```diff
@@ -111,6 +135,61 @@ const updateMessages = (id, path, req) => {
   )
 };
 
+const importMessages = (line) => {
+  const message = line.split(",");
+  const viewMessage = (message[1] === "Intro");
```

Comment:
Yeah, since "Intro" appears multiple times, it really seems like we should make it a constant.